frontend/src/components/TaskDetails/TaskDetails.css

```diff
@@ -0,0 +1,118 @@
+.taskDetails {
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  gap: 0.7rem;
+  background-color: var(--text--10);
+  margin: 5rem 0 2rem 0;
+  border-radius: 8px;
+}
+
+.details-icon {
+  width: 0.8rem;
+  height: 0.8rem;
+  color: #7d7a89;
+}
+
+.single-title {
+  font-family: 'Manrope', sans-serif;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 18px;
+  color: #2b283d;
+}
+
+/* Header */
+.task-details-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+/*  Rest */
+.other-details {
+  display: flex;
+  flex-direction: column;
+  gap: 0.8rem;
+}
+
+.single-detail {
+  display: flex;
+  flex-direction: column;
+  gap: 0.6rem;
+}
+
+.single-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.attachment-files {
+  display: flex;
+  align-items: center;
+  gap: 0.8rem;
+}
+
+.attachment-files img {
+  width: 4rem;
+  height: 4rem;
+  border-radius: 8px;
+}
+
+#small-check {
+  width: 1.2rem;
+  height: 1.2rem;
+  color: #5fb918;
+}
+
+#small-circle {
+  width: 1.2rem;
+  height: 1.2rem;
+  color: #b3b1bb;
+}
+
+.objects {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.object,
+.comment-section {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+}
+
```

Comment:
very generic names, i am sure you can come up with something a bit more descriptive :)

Reply:
I changed the names :)